app/routes.py

```diff
@@ -1,2 +1,225 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from sqlalchemy import desc
+from datetime import date
+import requests, os
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+''''''''''''''''''''''''''''''''''''
+# # # # # # TASK ROUTES # # # # # #
+''''''''''''''''''''''''''''''''''''
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def handle_tasks():
+    if request.method == "POST":
+        request_body = request.get_json()
+        title = request_body.get("title")
+        description = request_body.get("description")
+
+        if not title or not description or "completed_at" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
+
+        new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
```

Comment:
Because you defined title and description on lines 20/21 you can just use title and description on lines 26/27
For example: `Task(title = title)`